Sources/Xcodeproj/generate().swift

```diff
@@ -104,6 +104,12 @@ public func generate(
         extraDirs = []
     }
 
+    // If there is a project root README, include that in the root of the Xcodeproj
+    let readmePath = xcodeprojPath.parentDirectory.appending(component: "README.md")
+    if localFileSystem.exists(readmePath, followSymlink: false) {
+        extraFiles.append(readmePath)
```

Comment:
We need to check if extraFiles already contains the readme. See the code at line 94, we pick up non-gitignored files present in the repository. (But this is still useful if there is no git repository yet)

Reply:
Awesome, I was hoping that's what that was for. Is that code newer then `Apple Swift version 4.2.1 (swiftlang-1000.11.42 clang-1000.11.45.1)` - because that's not happening for me on the current prod build?

Reply:
Yes, the current master branch is targeting the Swift 5 release

Reply:
Perfect, OK, awesome - I'm going to close this PR. 

I think *not* using git is pretty edge-casey (given that git is baked into SwiftPM) and it's likely folks will need to regeneate the xcodeproj multiple times, giving them the readme. The existing behavior does what I'm looking for and I'm happy with that, good job 👍